Update GitHub authentication flow (#1055)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_managing-pull-requests-using-the-github-pr-plug-in.adoc

== Using the GitHub Pull Requests plug-in

.Prerequisites

> * GitHub OAuth should be configured (see https://www.eclipse.org/che/docs/che-7/version-control/#configuring-github-oauth_version-control[Configuring GitHub OAuth)]

Reviewer: ```suggestion
* GitHub OAuth should be configured (see link:https://www.eclipse.org/che/docs/che-7/version-control/#configuring-github-oauth_version-control[Configuring GitHub OAuth)].
```

.Procedure

 . Authenticate by running the *GitHub authenticate* command.
 . You will be redirected to GitHub to authorize Che.
 . When Che is authorized, refresh the browser page where Che is running to update the plug-in with the GitHub token.

> You can also manually https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line[fetch the GitHub token] and paste it to the plug-in by running *GitHub Pull Requests: Manually Provide Authentication Response*.

Reviewer: ```suggestion
Alternatively, manually link:https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line[fetch the GitHub token] and paste it to the plug-in by running the *GitHub Pull Requests: Manually Provide Authentication Response* command.
```
